src/docs/JobAnalytics/JobAnalyticsUserGuide.adoc

```diff
@@ -0,0 +1,96 @@
+- _Project Name(s)_: You can also filter by one or more project names. You have just to specify the project names for the jobs you would like to analyze.
+- _Job State(s)_: You can specify the state of jobs you are looking for. For instance, you can filter `Pending`, `Running`, `Stalled`, `Paused`, `In_Error`, `Finished`, `Canceled`, `Failed`, and `Killed` jobs. For more information about job states, check the documentation link:../user/ProActiveUserGuide.html#_states[here.] Multi value is accepted as well.
+- _User(s)_: This filter allows to either select only the user jobs or to specify the list of users that have executed the workflows. By default, the toggle filter is activated to select only the user jobs.
+- _Submission Time_: From the dropdown list, users can select jobs submitted today, yesterday, this week, last week, this month, last month, last month, or choose custom dates by specifying the begin and end dates.
+- _Variables and results_: It is possible to choose to display or not the workflow variables and results. When deactivated, the charts related to variables and results evolution/correlation will not be displayed in the dashboard.
+
+More advanced search options (_highlighted in advanced search hints_) could be applied to select the filter values. For instance, regular expressions could be applied to select more general values. For example, names that start with a specific string value are selected using `value*`. Other supported expressions are: `_\*value_` for _ends with_, `*value*` for _contains_, `!value` for _not equals_, and `!\*value*` for _not contains_.
+The clear button allows to clear all filter values and the search button will request jobs from the scheduler database according to the provided filter values.
+
+In the top side of the search window, a reminder of the selected filter values is displayed to show the filters that were considered during the latest search query.
+
+[[_JA-search-png]]
+image::JA-search.png[align=center]
+
+=== Execution Metrics
+As shown in the screenshot below, Job Analytics provides a summary of the most important job execution metrics. For instance, the dashboard shows:
+
+- A first panel to display the number of total jobs that correspond to the user query and filters. It also shows the ratio of successful jobs over the total number, and the number of jobs that are in progress and not yet finished.
+- A second summary panel to display the number of jobs with issues. We distinguish two types of issues: jobs that are finished but have encountered issues during their execution, and interrupted jobs that did not finish their execution and were stopped due to several kinds of issues such as insufficient resources, manual interruption, etc. So, we distinguish 4 types of interrupted states: `In-Error`, `Failed`, `Canceled`, and `Killed`.
```

Comment:
Also add that clicking on the issues types / charts filters the jobs table to show the corresponding jobs.